x-pack/plugin/security/qa/security-basic/src/javaRestTest/java/org/elasticsearch/xpack/security/SecurityWithBasicLicenseIT.java

```diff
@@ -222,6 +226,23 @@ private void assertAuthenticateWithApiKey(Tuple<String, String> keyAndId, boolea
         }
     }
 
+    private String createServiceAccountToken() throws IOException {
+        final Request request = new Request("POST", "_security/service/elastic/fleet-server/credential/token/api-token-1");
+        final Response response = adminClient().performRequest(request);
+        assertOK(response);
+        @SuppressWarnings("unchecked")
+        final Map<String, String> tokenMap = (Map<String, String>) responseAsMap(response).get("token");
+        return tokenMap.get("value");
```

Comment:
Technically... 

For future proofing you don't know whether this JSON object will always contain exclusively String values. It might, in the future, have boolean, numbers or even nested objects.
```suggestion
        final Map<String, ?> tokenMap = (Map<String, ?>) responseAsMap(response).get("token");
        return String.valueOf(tokenMap.get("value"));
```